engine/src/test/java/com/redhat/cloud/notifications/events/EventConsumerTest.java

```diff
@@ -186,10 +189,13 @@ void testValidPayloadWithoutOrgId() {
                 MESSAGE_ID_INVALID_COUNTER_NAME,
                 MESSAGE_ID_MISSING_COUNTER_NAME
         );
+        // The action org ID is set by OrgIdTranslator after the Kafka message is received.
+        action.setOrgId(DEFAULT_ORG_ID);
```

Comment:
This confuses me. Shouldn't this have happened already with the same value for orgId?

Reply:
The `action` object from this test is used twice:
- it is serialized and sent as a Kafka payload (with `org_id=null`)
- it is compared with the action (another instance) that is processed by `EndpointProcessor` (this one has its `org_id` field populated with `OrgIdTranslator`)

The line you highlighted is meant to verify that `OrgIdTranslator` populated the `org_id` as intended. Since the original action object from the test didn't have an `org_id` value, it has to be set here, before the check.

Reply:
This line will go away as soon as we're done with the orgId migration.

Reply:
Thanks for the explanation